src/shared/constants.js

```diff
@@ -60,3 +60,7 @@ export const tutorialModes = {
     // hint is shown if they press the "show answers" button
     hint: 'hint',
 };
+
+// the prefix used for our deeplink urls,
+// such as 'mapswipe://login/xxx'
+export const deeplinkPrefix = 'devmapswipe://';
```

Comment:
this should be `mapswipe://` for the production app, correct? how do we make sure we load the correct prefix?

Reply:
Yes, I was thinking of using a separate prefix so that we can continue using both dev and prod apps side by side on a single phone. How we get this into the code is still a bit unclear to me. Here is easy, since it's a runtime evaluation. But the same value needs to be hardcoded at build time in `Info.plist` and `AndroidManifest.xml`. Maybe we can do something similar to how we set the values in `debugInfo.json` at build time. We could set `devmapswipe` as the default value, so that developers don't need to worry about it, and at build time on github actions, if it's a prod build, we replace the value with `mapswipe` in the 3 places we need.